tests/test_hstrat/test_stratum_retention_strategy/test_stratum_retention_algorithms/test_stochastic_algo/test_PolicySpec.py

```diff
@@ -18,6 +19,16 @@ def test_eq(replicate):
     assert spec == stochastic_algo.PolicySpec()
 
 
+def test_GetEvalCtor():
+    spec = stochastic_algo.PolicySpec()
+    reconstituted = eval(spec.GetEvalCtor())
+    eval_ctor = spec.GetEvalCtor()
+    assert eval_ctor.startswith("hstrat.stochastic_algo.PolicySpec(")
+    assert eval_ctor.endswith(")")
+    reconstituted = eval(eval_ctor)
+    assert str(spec) == str(reconstituted)
```

Comment:
Why are we comparing str()’s, can we compare directly (I think there’s operator==?)

Reply:
There is! Changed them all in 722a513